src/02-common-principles.md

```diff
@@ -161,6 +161,27 @@ A summary of all entities in BIDS and the order in which they MUST be
 specified is available in the [entity table](./99-appendices/04-entity-table.md)
 in the appendix.
 
+### Entity-linked file collections
+
+A set of files that are related to each other based on a repetitive acquisition of
+sequential data by changing acquisition parameters one (or multiple) at a time and/or by
+being inherent components of the same data.
```

Comment:
Better to use a full sentence instead of a dictionary-like fragment:

```suggestion
An entity-linked file collection is a set of files that are related to each other
based on a repetitive acquisition of sequential data
by changing acquisition parameters one (or multiple) at a time
or by being inherent components of the same data.
```

I've dropped the `and/or` for `or`. I'm not sure being an explicitly inclusive "or" gains clarity.